### PR TITLE
Moves

### PR DESCRIPTION
**Pull request by teelu**
created code for moving the cup objects around.
PLEASE REVIEW CUP FUNCTION FIRST.  THANKS

### CODE REVIEW

File: js/application.js [hunk 1]
```diff
@@ -1,3 +1,81 @@
+function rand(limit) {
+  return Math.floor((Math.random()*(limit+1)));
+}
+//Game Class
+function Game() {
+  //variables
+  this.cupArray = [];
+  this.movesArray = [];
+  //cup constructor
```
Comment (f3r) [inline js/application.js]: line break between methods please, makes it easier to read :)

**Comment by teelu** [comment]
Added line breaks between methods :)

**Comment by h8rry** [comment]
#2 is merged. Please rebase this branch with master.

**Comment by teelu** [comment]
Thanks!

**Pull request merged**